### DISCUSSION TITLE
Couldn't load lyrics on Pi 2B / OSMC

### DISCUSSION THREAD

**Discussion by mountainhill2** [dtcooper/raspotify#549] [Q&A] [answered]
Hi, I get message "Couldn't load lyrics for this song" in Raspotify on Pi 2B with OSMC.
Does someone perhaps have a fix for this issue?
Thanks in advance.

**Comment by JasonLG1979** [comment, marked as answer]
Raspotify has nothing to do with lyrics.

**Reply by mountainhill2** [reply]
It was working previously before I updated to Bullseye. I thought other people might have experienced the same.

If I start in my phone and get the lyrics uploaded, I still get the lyrics working for that song even after I switch to raspotify. But it won't work again for the next song.

**Reply by JasonLG1979** [reply]
I can reproduce your bug. This out of the scope of Raspotify. Please file a bug with librespot. The issue may already be fixed though in the (relatively) soon to be merged `new-api` branch?

**Reply by mountainhill2** [reply]
Thank you for your pointer. Appreciate that.
